src/Features/Lsif/Generator/Program.cs

```diff
@@ -43,7 +44,19 @@ private static async Task GenerateAsync(FileInfo? solution, FileInfo? project, F
         {
             // If we have an output file, we'll write to that, else we'll use Console.Out
             using var outputFile = output != null ? new StreamWriter(output) : null;
-            var outputWriter = outputFile ?? Console.Out;
+            TextWriter? outputWriter = null;
+
+            if (outputFile is null)
+            {
+                Console.Out.Encoding = UTF8Encoding.UTF8;
+                Console.OutputEncoding = UTF8Encoding.UTF8;
+                Console.Error.Encoding = UTF8Encoding.UTF8;
```

Comment:
interesting.  why do we need to set OutputEncoding and Error.Encoding?  We're just writing to the outputWRiter right?  So i would expect we'd just need the first line.

Can you doc?

Reply:
Indeed, I wouldn't imagine we're messing with error encodings at all.

Reply:
Answered below, was a mistake, these aren't actually settable, errors just didn't come up fast enough (compile error)